### PR TITLE
default yaml,json handlers

### PR DESCRIPTION
**Pull request by dorschw**
replace default handlers with a constant

### CODE REVIEW

File: demisto_sdk/commands/init/templates/BaseIntegration/BaseIntegration_test.py [hunk 1]
```diff
@@ -12,7 +12,10 @@
 
 from demisto_sdk.commands.common.handlers import JSON_Handler
 
-json = JSON_Handler()
+from demisto_sdk.commands.common.handlers import (
+    DEFAULT_JSON_HANDLER as json,
+    DEFAULT_YAML_HANDLER as yaml,
```
Comment (kobymeir) [inline demisto_sdk/commands/init/templates/BaseIntegration/BaseIntegration_test.py:17]: no need for yaml?

File: demisto_sdk/commands/init/templates/BaseScript/BaseScript_test.py [hunk 2]
```diff
@@ -10,7 +10,10 @@
 
 from demisto_sdk.commands.common.handlers import JSON_Handler
 
-json = JSON_Handler()
+from demisto_sdk.commands.common.handlers import (
+    DEFAULT_JSON_HANDLER as json,
+    DEFAULT_YAML_HANDLER as yaml,
```
Comment (kobymeir) [inline demisto_sdk/commands/init/templates/BaseScript/BaseScript_test.py:15]: no need for yaml?

File: demisto_sdk/commands/generate_test_playbook/test_playbook_generator.py [hunk 3]
```diff
@@ -2,14 +2,12 @@
 from typing import Dict, Optional
 
 from demisto_sdk.commands.common.constants import FileType
-from demisto_sdk.commands.common.handlers import JSON_Handler, YAML_Handler
+from demisto_sdk.commands.common.handlers import DEFAULT_JSON_HANDLER as json
+from demisto_sdk.commands.common.handlers import DEFAULT_YAML_HANDLER as yaml
```
Comment (kobymeir) [inline demisto_sdk/commands/generate_test_playbook/test_playbook_generator.py:6]: join imports

**Review by kobymeir — CHANGES_REQUESTED** [review]

File: demisto_sdk/commands/generate_integration/code_generator.py [hunk 4]
```diff
@@ -28,7 +29,6 @@
 )
 from demisto_sdk.commands.generate_integration.XSOARIntegration import XSOARIntegration
 
-json = JSON_Handler()
 yaml = YAML_Handler(width=50000)
```
[the remark this answers is not in the document]
Comment (dorschw) [reply demisto_sdk/commands/generate_integration/code_generator.py:32]: different issue

File: demisto_sdk/commands/generate_test_playbook/test_playbook_generator.py (hunk 3 above)
Comment (dorschw) [reply demisto_sdk/commands/generate_test_playbook/test_playbook_generator.py:6]: ruff I001 says no

File: demisto_sdk/commands/generate_integration/code_generator.py [hunk 5]
```diff
@@ -8,7 +8,8 @@
 
 import demisto_sdk.commands.common.tools as tools
 from demisto_sdk.commands.common.constants import ParameterType
-from demisto_sdk.commands.common.handlers import JSON_Handler, YAML_Handler
+from demisto_sdk.commands.common.handlers import DEFAULT_JSON_HANDLER as json
```
[the remark this answers is not in the document]
Comment (dorschw) [reply demisto_sdk/commands/generate_integration/code_generator.py:11]: ruff I001 says no

File: demisto_sdk/commands/pre_commit/tests/pre_commit_test.py [hunk 6]
```diff
@@ -8,10 +8,10 @@
 from demisto_sdk.commands.pre_commit.hooks.ruff import RuffHook
 from demisto_sdk.commands.pre_commit.pre_commit_command import (
     GitUtil,
-    YAML_Handler,
     group_by_python_version,
     preprocess_files,
     subprocess,
+    yaml,
```
[the remark this answers is not in the document]
Comment (dorschw) [reply demisto_sdk/commands/pre_commit/tests/pre_commit_test.py:14]: here it's by design, as it's only for mocking.

File: demisto_sdk/commands/common/tools.py [hunk 7]
```diff
@@ -115,15 +115,17 @@
 )
 from demisto_sdk.commands.common.git_content_config import GitContentConfig, GitProvider
 from demisto_sdk.commands.common.git_util import GitUtil
-from demisto_sdk.commands.common.handlers import JSON_Handler, YAML_Handler
+from demisto_sdk.commands.common.handlers import (
+    DEFAULT_JSON_HANDLER as json,
+    DEFAULT_YAML_HANDLER as yaml,
+    YAML_Handler,
+)
```
[the remark this answers is not in the document]
Comment (dorschw) [reply demisto_sdk/commands/common/tools.py:122]: Asked them [here](https://github.com/astral-sh/ruff/issues/5836)

File: demisto_sdk/commands/content_graph/objects/pack.py [hunk 8]
```diff
@@ -83,7 +82,7 @@
 if TYPE_CHECKING:
     from demisto_sdk.commands.content_graph.objects.relationship import RelationshipData
 
-json = JSON_Handler()
+from demisto_sdk.commands.common.handlers import DEFAULT_JSON_HANDLER as json
```
Comment (kobymeir) [inline demisto_sdk/commands/content_graph/objects/pack.py:85]: it was on line 19 before, now on 85. why not move the import up before the if statement

**Review by kobymeir — CHANGES_REQUESTED** [review]

**Review by kobymeir — APPROVED** [review]

**Pull request merged**